src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  box-sizing: content-box;
+  margin: 0;
+}
+
+.Search {
+  height: 70px;
+  width: 100vw;
```

Comment:
Dont set width to 100vw (it is usually a bad practice) - this way you might end up with unneeded scroll area.
By default block elements will take 100% of its container so there is no need for setting width